packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx

```diff
@@ -11,4 +11,8 @@ const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
 const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
 const displays = [display1, display2, display3, display4, display5, display6];
 
-export default () => <ItemView displays={displays} layout="twoColumns" />;
+export default () => (
+  <div>
```

Comment:
Do we need this extra div?

Reply:
updated here https://github.com/cerner/terra-clinical/pull/630/commits/e0598c8feb11116a5aca962c2d695312dfa5e79c.